subiquity/models/filesystem.py

```diff
@@ -150,16 +223,49 @@ def from_info(self, info):
         d.model = info.model
         return d
 
+    def info_for_display(self):
+        bus = self._info.raw.get('ID_BUS', None)
+        major = self._info.raw.get('MAJOR', None)
+        if bus is None and major == '253':
+            bus = 'virtio'
+
+        devpath = self._info.raw.get('DEVPATH', self.path)
+        # XXX probert should be doing this!!
```

Comment:
Yes, I thought udev already had this property ?  No, no it doesn't.  Can this be added to the probe method so we fetch that on creation? it won't change over the life of the object.

Reply:
ditto